dotcom-rendering/src/web/layouts/lib/newsletterSignupLayoutUsage.tsx

```diff
@@ -0,0 +1,28 @@
+import { ArticleDesign, ArticleDisplay } from '@guardian/libs';
+import { decidePalette } from '../../lib/decidePalette';
+import { NewsletterSignupLayout } from '../NewsletterSignupLayout';
+
+export const hasNewsletterSignUpTag = (CAPIArticle: CAPIArticleType): boolean =>
+	CAPIArticle.tags.some((tag) => tag.id === 'info/newsletter-sign-up');
+
+export const buildNewsletterSignupLayout = (
```

Comment:
Is this a function used for local testing? If so, can I suggest we replace it with a story? Storybook is a really great tool for this kind of work, I think you will end up loving it if you give it a chance 😉 

Very happy to pair on getting this setup

Reply:
Thanks, I've added a story with a few variations for the layout and removed this file

Reply:
We should catch up about this. I think if we just add the [version of format](https://github.com/guardian/libs/blob/main/src/format/ArticleDesign.ts) that contains `NewsletterSignup` then you should automatically just get stories (including all the variation with different themes). I will chase up adding this version for you

Reply:
Oh, so actually, we already have the `NewsletterSignup` design type, it was added recently!

If you look at [this Chromatic page](https://www.chromatic.com/component?appId=5dfcbf3012392c0020e7140b&csfId=layouts-format-variations-standard-newslettersignup&buildNumber=12558&k=627e22c3fe2deb004a36404f-1200-interactive-true&h=13&b=-1) you can see the way the page currently looks.